packages/manager/apps/dedicated/package.json

```diff
@@ -125,7 +125,7 @@
     "oclazyload": "^1.1.0",
     "office-ui-fabric-core": "^11.0.0",
     "ovh-angular-responsive-tabs": "^4.0.0",
-    "ovh-api-services": "^9.38.0",
+    "ovh-api-services": "^9.39.0",
```

Comment:
This should be done for all packages using `ovh-api-services`
(`yarn upgrade-interactive --latest ovh-api-services`)

Reply:
Like @FredericEspiau said, it can be done at the end. Thanks for pointing this out @marie-j  :)

Reply:
Solved after the rebase from develop